### DISCUSSION TITLE
How to make a post list page with last updated time?

### DISCUSSION THREAD

**Discussion by StreakingMan** [vuejs/vitepress#2839] [Q&A] [answered]
I have read the docs, createContentLoader is a good way, and I found this code in docs:

``` javascript
transform(rawData) {
    // map, sort, or filter the raw data as you wish.
    // the final result is what will be shipped to the client.
    return rawData.sort((a, b) => {
      return +new Date(b.frontmatter.date) - +new Date(a.frontmatter.date)
    }).map((page) => {
      page.src     // raw markdown source
      page.html    // rendered full page HTML
      page.excerpt // rendered excerpt HTML (content above first `---`)
      return {/* ... */}
    })
  }
```

but sometimes we don't manually write date in frontmatter，is there some way to get all posts' last updated time just like runtime API?

**Comment by brc-dd** [comment, marked as answer]
For the moment, you can copy getGitTimestamp function from our source code and use it inside transform. We can expose that if there is much demand.

**Reply by StreakingMan** [reply]
❤️  Thanks a lot! This is very usefullfor blog-like websites, really hope it can be exposed. (BTW, I have found a related discussion https://github.com/vuejs/vitepress/discussions/2459#discussioncomment-6056965)
